renderer/shaders.cmake

```diff
@@ -0,0 +1,31 @@
+file(WRITE ${shaders_h} "#pragma once
```

Comment:
Probably better move this file to top-level `cmake` directory where our cmake functions resides.